web/src/components/text-input/styles.ts

```diff
@@ -1,92 +1,190 @@
 import { createUseStyles } from 'react-jss'
 
 import tokens from '../../tokens'
-import { getColorDefault, getColorHover } from '../checkbox/styles'
+import { parseToken } from '../../utils'
+import { getColorDefault } from '../checkbox/styles'
+import customScrollbar from '../scrollbar/styles'
 
-export default createUseStyles({
+const textAreaHeight = 61
```

Comment:
not a token?

Reply:
Such things we're not defining as tokens, anyway it fits now for current size tokens, but we don't know how it will work with sizes x1.5 and what value I should put here, it's not so easy, as like multiply it to 1.5 and that's it. Unfortunately not